src/Expectation.php

```diff
@@ -715,6 +716,53 @@ public function toMatchConstraint(Constraint $constraint): Expectation
         return $this;
     }
 
+    /**
+     * Asserts that executing value throws an exception.
+     *
+     * @param string|\Closure $exception string: the exception class
+     *                                   Closure: first parameter = exception class
+     */
+    public function toThrow($exception): Expectation
```

Comment:
I am sorry, but this is missing the same API as `throws` where you can specify the Exception class + the message.